bootcamp/tutorials/integration/llama_agents_metadata.ipynb

```diff
@@ -0,0 +1,1130 @@
+{
```

Comment:
<p>The above code block has good explanation, but this block misses such things. E.g. link to ReAct as i mentioned in another comment, and how the two tools are playing here and why missing metadata filtering is bad.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1401/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>